src/Plugin/Autocomplete/Handler.php

```diff
@@ -71,6 +73,29 @@ public function run(): Task {
 		return $task->run();
 	}
 
+	/**
+	 * Perform validation that is should be run in coroutine
+	 * @return void
+	 */
+	protected function validate(): void {
+		$cacheKey = "autocomplete:validate:{$this->payload->table}";
+		if ($this->flagCache->has($cacheKey)) {
+			return;
+		}
+		/** @var array{error?:string} */
+		$result = $this->manticoreClient->sendRequest('SHOW CREATE TABLE ' . $this->payload->table)->getResult();
+		if (isset($result['error'])) {
```

Comment:
Should there be QueryValidationError? Cause the reason can be pretty wide from connection error, to absent table

Reply:
In case of a connection error, we received another exception from the Manticore Client. This likely means the issue is only about validation.